cloud/misc/proxmox_kvm.py

```diff
@@ -745,7 +758,11 @@ def create_vm(module, proxmox, vmid, node, name, memory, cpu, cores, sockets, ti
   if module.params['api_user'] != "root@pam" and module.params['skiplock'] is not None:
     module.fail_json(msg='skiplock parameter require root@pam user. ')
 
-  taskid = getattr(proxmox_node, VZ_TYPE).create(vmid=vmid, name=name, memory=memory, cpu=cpu, cores=cores, sockets=sockets, **kwargs)
+  if module.params['clone'] == 'None':
```

Comment:
right ! but `clone` have some args not common with `state: present` (newid, vmid, description, format, full, pool, snapname, storage, target).

Your approach, of course, makes sense. We can start on this basis, I would make some improvements and I will submit it for discussion.